packages/cli/src/commands/container/pull.ts

```diff
@@ -1,8 +1,10 @@
 import {Command, flags} from '@heroku-cli/command'
 import {ux} from '@oclif/core'
 import * as DockerHelper from '../../lib/container/docker_helper'
+import * as helpers from '../../lib/container/helpers'
```

Comment:
Could you just import the `ensureContainerStack` function here instead of the whole file?